tests/test_available_tasks.py

```diff
@@ -0,0 +1,26 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import pytest
+
+from pages.home import HomePage
+
+
+class TestAvailableTasks():
+
+    @pytest.mark.nondestructive
+    def test_available_tasks(self, base_url, selenium):
+        home_page = HomePage(base_url, selenium).open()
+        assert not home_page.is_user_logged_in
+
+        available_tasks_page = home_page.click_available_tasks()
+        assert available_tasks_page.is_the_current_page
```

Comment:
We don't need this assert because `wait_for_page_to_load` calls `is_the_current_page` and `home_page.click_available_tasks()` calls `wait_for_page_to_load`.